$OpenBSD: patch-browser_installer_allowed-dupes_mn,v 1.1 2017/03/07 15:36:45 landry Exp $

Needed otherwise mozilla packaging code complains on those dupes
--with-app-name should match those

--- browser/installer/allowed-dupes.mn.orig	Thu Mar  2 22:54:58 2017
+++ browser/installer/allowed-dupes.mn	Thu Mar  2 22:55:07 2017
@@ -209,8 +209,8 @@ components/FxAccountsPush.js
 crashreporter.app/Contents/Resources/English.lproj/MainMenu.nib/classes.nib
 crashreporter.app/Contents/Resources/English.lproj/MainMenuRTL.nib/classes.nib
 # firefox/firefox-bin is bug 658850
-firefox
-firefox-bin
+firefox-esr
+firefox-esr-bin
 modules/FxAccountsPush.js
 modules/commonjs/index.js
 modules/commonjs/sdk/ui/button/view/events.js
